apps/e2e/tests/backend/endpoints/api/v1/auth/sessions/index.test.ts

```
@@ -100,8 +100,8 @@ it("creates sessions that expire", async ({ expect }) => {
     await Auth.expectToBeSignedIn();
   } finally {
     const timeSinceBeginDate = new Date().getTime() - beginDate.getTime();
-    if (timeSinceBeginDate > 4_000) {
-      throw new StackAssertionError(`Timeout error: Requests were too slow (${timeSinceBeginDate}ms > 4000ms); try again or try to understand why they were slow.`);
+    if (timeSinceBeginDate > 6_000) {
+      throw new StackAssertionError(`Timeout error: Requests were too slow (${timeSinceBeginDate}ms > 6000ms); try again or try to understand why they were slow.`);
```

Biome (2.3.13)

[error] Unsafe usage of 'throw'.

'throw' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)